rosetta-source/src/main/resources/cdm-sample-files/functions/repo-and-bond/partial-delivery.md

```diff
@@ -0,0 +1,11 @@
+### Create Partial Delivery Business Event
+
+#### New Execution
+- Execution is performed on `2021-3-18` between parties `Global Bank Inc (Seller)` and `UK Bank plc (Buyer)`
+- `10000000` notional of `GB00B24FF097` are sold with dirty price of `1.0075 GBP`, a repo rate of `0.4%` and haircut of `1%`
+- Purchase date is on `2021-3-19` and repurchase date `2021-3-22`
+
+#### Partial Delivery Event
+- On the day of the settlement on `2021-3-18`, the seller delivers `750000` notional of `GB00B24FF097`. 
+- Even though this delivery does not fulfill the required delivery amount the buy accepts the amount the trand is adjusted accordingly.
```

Comment:
... **buyer** accepts the amount **and the trade** is...

Reply:
fixed